### ISSUE TITLE
[MStore Pro] issue with android in v 3.7.1

### ISSUE THREAD

**Issue by yosefasad** [inspireui/support#398]
Hello 
I installed the new version of Mstore pro 3.7..1 and have issue with android build. please see the video and screen shot. 

https://youtu.be/Gs-tpYKn6tM
https://imgur.com/a/dNrhrek

need help

**Comment by levi-beers** [comment]
@yosefasad I have no issues building MStore Pro 3.7.1 for Android. Make sure you are using Android Build Tools 28.0.3. Use Android Studio or IntellijIDEA to automatically get the proper version of gradle & build tools, then you can use CLI to assemble release APK. 
<img width="1041" alt="Screen Shot 2019-03-13 at 11 43 40 PM" src="https://user-images.githubusercontent.com/16378407/54329927-37a90b80-45ea-11e9-8377-94a2e64ec179.png">

**Comment by yosefasad** [comment]
it is already  Android Build Tools 28.0.3.

please check the video
https://youtu.be/XcUkaIe2818

**Comment by minhcasi** [comment]
@yosefasad - 
it seems you did not select:
Android SDK Platform 28
Intel x86 Atom_64 System Image or Google APIs Intel x86 Atom System Image

https://facebook.github.io/react-native/docs/getting-started.html

**Comment by yosefasad** [comment]
these are my config 
https://youtu.be/3XlNgR3LEjg

I already build MStore 3.7.0 and work perfectly .  please see . the video
https://youtu.be/FLFyoRKUjNc

still it is not working well

**Comment by minhcasi** [comment]
@yosefasad - thanks for the detailed video, i will send to our team for investigating

Could i know which the Android OS simulator that you are testing, please try to difference simulator to see if can work fine ?

**Comment by yosefasad** [comment]
hello
I changed the simulator , still same issue, ... please take a look at 
https://imgur.com/a/tLWbsKy

the simulator i used before 
https://imgur.com/a/5qvcJwU

**Comment by luyx2412** [comment]
Hi @yosefasad I think something was wrong about your config, Can you share AppConfig.json, Config.js and Constants.js file?

Thanks

**Comment by yosefasad** [comment]
They re the default version from download zip file
[Archive.zip](https://github.com/inspireui/support/files/2977539/Archive.zip)

**Comment by luyx2412** [comment]
Hi @yosefasad I just checked your config, everything worked normal. Please open your android studio, and check specific error based on this article https://stackoverflow.com/a/16876993/6622971

Thanks

**Comment by yosefasad** [comment]
Thank you for your reply

I think the solution in this  article https://stackoverflow.com/a/16876993/6622971   is little outdated and I did not know how to apply the solution where the Android Studio interface in the solution is old. 

I  record a .video to show you all the steps and configuration is made to make this app work ...and the issue still exists :(

https://youtu.be/UxmPXN-dRR4

I hope you help me to find the miss configuration - or solution .....

Thank you again for your help and reply.



<blockquote><img src="https://cdn.sstatic.net/Sites/stackoverflow/img/apple-touch-icon@2.png?v=73d79a89bded" width="48" align="right"><div>Stack Overflow</div><div><strong><a href="https://stackoverflow.com/questions/16633956/android-studio-where-is-the-compiler-error-output-window/16876993">Android Studio: Where is the Compiler Error Output Window?</a></strong></div><div>When I 'Run' my project in Android Studio, in the 'Messages' window, I get:

Gradle: 
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':play01:compileDebug'.
...</div></blockquote>
<blockquote><img src="https://i.ytimg.com/vi/UxmPXN-dRR4/hqdefault.jpg" width="48" align="right"><div><img src="/yts/img/favicon_144-vfliLAfaB.png" height="14"> YouTube</div><div><strong><a href="https://www.youtube.com/watch?v=UxmPXN-dRR4">all config</a></strong></div><div> </div></blockquote>

**Comment by luyx2412** [comment]
Hi @yosefasad I think the issue related the data, Can you use debug tools to check the api response? https://docs.mstore.io/api-and-project-structure#debug-app-with-reacttotron
And are you change code in the project? Can you give me the code you edited or send me your project?

Thanks

**Comment by yosefasad** [comment]
Hello

the code is the same as downloaded from envato , I have not change any code , it is the default .
this the project 

https://drive.google.com/file/d/1oPZLLGz72vDVbzICPpzNUzfXzxmTCqLv/view?usp=sharing


today I reinstall MAC OS , and I got this error 

The development server returned response error code: 500 URL: http://10.0.2.2:8081/index.delta?platform=android&dev=true&minify=fail

https://imgur.com/a/ICPcFqb

and this is my initial environment
https://imgur.com/a/zeqIkHW

Also, I tried the project with different emulators, some Emulator did not work, give the this Error the "emulator is OFFLINE "
Please advice which emulator you recommend to use? 
Also, There is nothing is shown on Reactotron    

***** . I hope this help, any suggestion ?? 


Thank you for support

**Comment by yosefasad** [comment]
Hello

Please check this screenshoot https://imgur.com/a/PcghUs7
Thanks

**Comment by minhcasi** [comment]
@yosefasad - just remind that we have deleted your comment as it is contented the source code that is selling from our marketplace - you can send the email to support@inspireui.com if there is content any private info - thanks

**Comment by yosefasad** [comment]
Dear minhcasi

I apology for the mistake, any private info will be sent to you via email - next time  

thanks

**Comment by binaryversion** [comment]
Did anyone manage to solve the issue? I'm having this problem just with Android, on IOs works perfectly.

**Comment by yosefasad** [comment]
it is about your Android studio configuration.  try to make your SDK Platform to be android  8.0 oreo. It will work with you.  

Good luck

**Comment by binaryversion** [comment]
Hi think it is already with this:
buildscript {

    ext {

        buildToolsVersion = "28.0.3"
        minSdkVersion = 21
        compileSdkVersion = 28
        targetSdkVersion = 28
        supportLibVersion = "28.0.0"
    }
    repositories {

        google()

        jcenter()

    }

    dependencies {

        classpath 'com.google.gms:google-services:3.0.0'
        classpath 'com.android.tools.build:gradle:3.4.0'

        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
}








Atentamente | Kind regards | Mit freundlichen Grüßen,



João Martins





--

Este  e-mail é estritamente confidencial e para uso somente pelo (s)  destinatário (s) pretendido (s). Todas as opiniões expressas são as do  autor. Portanto, o seu conteúdo não representa qualquer compromisso  entre Binaryversion, Lda e o (s) destinatário (s), e nenhuma  responsabilidade será aceite pela Binaryversion, Lda pelo conteúdo acima  mencionado. Se você não for um destinatário pretendido, notifique a  info arroba binaryversion ponto pt prontamente e destrua esta mensagem.

--

This  e-mail is strictly confidential and for use only by the intended  recipient(s). Any opinions therein expressed are those of the  author. Therefore its content does not represent any commitment  between Binaryversion, Lda and the recipient(s), and no liability or  responsibility is accepted by Binaryversion, Lda for the above-mentioned  content. If you are not an intended recipient, please notify info  at binaryversion dot pt promptly and destroy this message.

--








---- On Tue, 23 Apr 2019 21:04:34 +0100 yosefasad <mailto:notifications@github.com> wrote ----



it is about your Android studio configuration.  try to make your SDK Platform to be android  8.0 oreo. It will work with you.

Good luck

**Comment by minhcasi** [comment]
@binaryversion - did you follow this guide to reinstall the Android SDK - https://facebook.github.io/react-native/docs/getting-started.html (select React Native CLI) and is it possible to run the example project from your setting ?

**Comment by binaryversion** [comment]
@minhcasi - I did reinstalled everything and restart from scratch. I unziped the files from the MStore PRO packadge, and followed the Manual Documentation. I didn't change any of the original code and on IOS works perfectly with Android doesn't work *1.
*1 Means that the APP open with an error and every time I try to get Categories ou any other service with API with online query the Error comes UP.

**Comment by yosefasad** [comment]
@binaryversion 
I had  the same issue, the issue was solved by going to AVD Manager in Android Studio, and create new virtual Device running on Oreo API level 27.    the issue was resolved.

Otherwise the issue still exist for Release Pie , level 28. ....I could not get rid of the error. even If I did not change any code, running on 3.7.2 , still the issue is there ....  

I hope it will be solved soon.
